builtin/credential/okta/backend.go

```diff
@@ -115,9 +128,81 @@ func (b *backend) Login(ctx context.Context, req *logical.Request, username stri
 	case "PASSWORD_WARN":
 		oktaResponse.AddWarning("Your Okta password is in warning state and needs to be changed soon.")
 
-	case "MFA_REQUIRED", "MFA_ENROLL":
+	case "MFA_ENROLL", "MFA_ENROLL_ACTIVATE":
 		if !cfg.BypassOktaMFA {
-			return nil, logical.ErrorResponse("okta mfa required for this account but mfa bypass not set in config"), nil, nil
+			if b.Logger().IsDebug() {
+				b.Logger().Debug("auth/okta: user must enroll or complete mfa enrollment", "user", username)
+			}
+			return nil, logical.ErrorResponse("okta authentication failed: you must complete MFA enrollment to continue"), nil, nil
+		}
+
+	case "MFA_REQUIRED":
+		if cfg.BypassOktaMFA {
+			result.Status = "SUCCESS"
```

Comment:
Can you add a code comment here explaining that without completing MFA we can't know any other status, but if we get to this point the username/password is valid and account is not locked out?